core/Enclave/enclave_hsm.edl

```diff
@@ -105,6 +105,8 @@ enclave {
                                             [out, size=plaintext_len] uint8_t *plaintext, uint32_t plaintext_len,
                                             [out] uint32_t *req_plaintext_len);
 
+        public sgx_status_t enclave_generate_apikey([out, size=apikey_len] uint8_t *p_apikey,
+                                                uint32_t apikey_len, [out] uint32_t *p_req_apikey_len);
```

Comment:
don't need [out] uint32_t *p_req_apikey_len

Reply:
[out] is required for pointers. 
not sure the default is [user_check] or [out]

Reply:
It will be change